app/components/HoverPreview/HoverPreview.tsx

```diff
@@ -111,7 +111,7 @@ function HoverPreviewInternal({ element, onClose }: Props) {
     : anchorBounds.left;
   const leftOffset = anchorBounds.left - left;
 
-  if (!data && loading) {
+  if (loading) {
```

Comment:
Switching among adjacent elements flashes previously rendered card just before rendering the correct one. Preventing that here.

https://github.com/outline/outline/assets/15694566/4033d563-ab94-4e51-8e6a-6a888c8fcc57